src/Domain/ReadySession/Model/ReadySessionTimestamp.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StepupReadId\Domain\ReadySession\Model;
+
+use Webmozart\Assert\Assert;
```

Comment:
See my previous comment about using third party libraries in the domain layer

Reply:
In this case yes... I was a little lazy here, I can create a guard because the code is simple.